src/main/java/com/lowagie/text/rtf/field/RtfTableOfContents.java

```diff
@@ -75,11 +75,11 @@
 	/**
 	 * field inst content
 	 */
-	private final static String FIELD_INST = "TOC \\\\f \\\\h \\\\u \\\\o \"1-5\" ";
+    private static final String FIELD_INST = "TOC \\\\f \\\\h \\\\u \\\\o \"1-5\" ";
     /**
      * The default text to display
      */
-    private String defaultText = "Table of Contents - Click to update";
+    private final String defaultText;
```

Comment:
Isn't the value of `defaultText` missing?

Reply:
This is the only question I have, that needs an answer.

Reply:
This field is always initialized in the sole constructor, so this value gets overwritten anyway.